x/circuit/features/msg_reset.feature

```diff
@@ -0,0 +1,55 @@
+Feature: MsgResetCircuitBreaker
+	- Circuit breaker can be reset:
+	- when the permissions are valid
+
+  Rule: caller with must have a permission to reset the circuit
+
+    Example: caller attempts to reset a disabled message
+      Given "acct1" has permission "LEVEL_SUPER_ADMIN"
+      When "acct1" attempts to enable a disabled message
```

Comment:
What happens when one attempts to enable an already enabled message? Error or no-op?

Reply:
Same to trip an already tripped msg

Reply:
we can error that is already done. I can add a case for this